helpers/build_packages.sh

```diff
@@ -187,7 +202,11 @@ if [ "$BUILDPKG" == "1" ]; then
     if [ -z $BUILDPKG_PATH ]; then
        echo "--build requires an argument (path to package)"
     else
-        buildpkg $BUILDPKG_PATH $BUILDSPEC_FILE
+        if [[ -z "$BUILDSPEC_FILE" ]]; then
+            buildpkg $BUILDPKG_PATH
+        else
+            buildpkg $BUILDPKG_PATH "$BUILDSPEC_FILE"
```

Comment:
...as this should suffice (previously it would pass only the first argument):
`buildpkg $BUILDPKG_PATH ${BUILDSPEC_FILE[@]}`

This https://github.com/mer-hybris/droid-hal-device/pull/205/files?utf8=%E2%9C%93&diff=unified#diff-748fa8f198703f775c5df5dc1fa219ccR295 then should be corrected as:
```
shift
build $@
```

Reply:
Good catch, I've incorporated your suggestion now.